fumadocs/content/docs/advanced/grid-suggestion-menus.mdx

```diff
@@ -14,9 +14,17 @@ The Emoji Picker is a Grid Suggestion Menu that opens with the `:` character (or
 
 It only displays once the user types 2 non-whitespace characters a query, to minimize cases where the user only wants to enter the `:` character.
 
-<ThemedImage
+<Image
+  className="block dark:hidden"
   src="/img/screenshots/emoji_picker.png"
-  darkImage="/img/screenshots/emoji_picker_dark.png"
+  alt="image"
+  width={400}
+  height={400}
+/>
+
+<Image
+  className="hidden dark:block"
+  src="/img/screenshots/emoji_picker_dark.png"
```

Comment:
Do we want to put two images everytime? Does the browser load both? Can this be abstracted to a ThemedImage that does this under the hood?